releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -1,12 +1,13 @@
 ## Azure App Configuration Extension release notes
 
-The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed.: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
+The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
 ### 1.6.3 - June, 03 2019
```

Comment:
Would it be clearer to add `Version` or `v` in front of the number, eg., `v1.6.3`?